frappe/handler.py

```diff
@@ -250,7 +250,7 @@ def run_doc_method(method, docs=None, dt=None, dn=None, arg=None, args=None):
 	try:
 		args = json.loads(args)
 	except ValueError:
-		args = args
+		pass
```

Comment:
- why not use `frappe.parse_json` here, since it already checks for string and dict to avoid the valueerror

Reply:
@hrwX good idea, but we still need to check for ValueError, as args might contain a simple string argument that should be conserved.